src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/BaseDrawioWebView.kt

```diff
@@ -105,6 +103,13 @@ abstract class BaseDrawioWebView(val lifetime: Lifetime) {
 
     private var requestId = 0
 
+    public fun reload(uiTheme: String) {
+        this.uiTheme = uiTheme
+        //initializeSchemeHandler(uiTheme)
```

Comment:
Why is this out commented?

Reply:
also part of the WIP. I though this would help but it didn't

Reply:
I was wrong - it is helpful. re-added it.